RedHat-UX/red-hat-design-system
# docs: fixed the personalization page images

## What I did

1. Updated the images for the "Personalization patterns" page


## Testing Instructions

1. Visit https://deploy-preview-1164--red-hat-design-system.netlify.app/foundations/personalization
2. Verify that the images match the [mockup](https://xd.adobe.com/view/ee0eb300-497d-4a49-8b6d-5dec9e9cfcbf-bb5d/)
3. Verify that the Patterns/Feedback sections on the bottom of the page match the mockup.
